src/Legolas.jl

```diff
@@ -1,6 +1,6 @@
 module Legolas
 
-using Tables, Arrow, UUIDs
+using Tables, Arrow, UUIDs, ConstructionBase
```

Comment:
I'm very much for being explicit about imports in libraries:
```suggestion
using Tables, Arrow, UUIDs
using ConstructionBase: ConstructionBase
```

Reply:
makes sense, especially since we're only defining methods, not calling anything AFAICT